README.md

```diff
@@ -1,4 +1,13 @@
-# React boilerplate
+# :zap Template Navy React
+
+This template has been created for the use of the AMAZING [https://github.com/lightning-resume/lightning-resume]lightning-resume project
```

Comment:
I believe you wanted the links to show in a different way, please take a look [here](https://github.com/lightning-resume/template-navy-react/tree/2/Update-readme)

You should do something like: [my text](https://blabla.com \)